rgf/sklearn.py

```diff
@@ -6,12 +6,18 @@
 from glob import glob
 import numpy as np
 from math import ceil
+import numbers
 
 from sklearn.base import BaseEstimator
 from sklearn.base import ClassifierMixin
 from sklearn.base import RegressorMixin
 from sklearn.utils.extmath import softmax
 from sklearn.utils.validation import NotFittedError
+from sklearn.externals import six
+
+_ALGORITHMS = ["RGF", "RGF_Opt", "RGF_Sib"]
+_LOSSES = ["LS", "Expo", "Log"]
+_FLOATS = (float, np.float, np.float16, np.float32, np.float64, np.double)
 
 sys_name = platform.system()
 WINDOWS = 'Windows'
```

Comment:
I think style of global constant should be unified.
Are there any reason to use `_LOSSES` (start with underscore) and `WINDOWS` (not start with underscore)?